enhancements/cloud-integration/tokenized-auth-enablement-operators-on-cloud.md

```diff
@@ -0,0 +1,493 @@
+follow instructions (for example [1]) to provide STS-based credentials to the operator such that needed resources can be
+accessed by the operand. This enhancement seeks to normalize this process across operators, so users of several of them
+have the same experience and similar steps to perform. We aim to automate and reduce those steps as much as possible to 
+make adding operators (via OLM) simpler on an STS enabled cluster.
+
+### User Stories
+
+* As a cluster admin I know which OLM Operators support tokenized authentication for my cloud.
+* As a cluster admin of a cluster using tokenized cloud auth I know what's required to install and upgrade OLM Operators whenever those operators manage resources that authenticate against my cloud.
+* As an Operator developer, I have a standard framework to define tokenized authentication requirements and consume them, per supported cloud.
+* As an Operator Hub browser I know which operators support tokenized cloud auth and on which clouds.
+* As the Hypershift team, where the only credential mode for clusters/customers is STS (on AWS) , the Red Hat branded
+  Operators that must reach the AWS API, should be enabled to work with STS credentials in a consistent, and automated
+  fashion that allows customer to use those operators as easily as possible, driving the use of layered products.
+
+### Goals
+
+Allow Day-2 Operators to access cloud provider resources as seamlessly as possible when being installed, used and
+updated on STS enabled clusters.
+
+While providing the above goal, allow for multi-tenancy. In this sense multi-tenancy means that an operator may
+provide different cloud provider credentials per operand such that operator users may uniquely access non-shared cloud
+resources.
+
+Operator authors have a way to notify, guide, and assist Day-2 Operator admins in providing the required cloud provider
+credentials matched to their permission needs for install and update.
+
+Ideally, a solution here will work in both HyperShift (STS always) and non-HyperShift clusters.
+
+### Non-Goals
+
+Day 1 operators are not included in this proposal, they are pre-provisioned for STS enabled clusters (at least 
+for HyperShift)
+
+
+
+## Proposal
+
+**Cloud Credential Operator (CCO) changes**: Adds a token-aware mode, while nominally set in "Manual" mode, CCO will
+look for and process CredentialRequests referenced in Operator CRs when:
+
+```bash
+# infrastructure platform is AWS
+$ oc get infrastructure cluster -o jsonpath --template '{ .status.platform }'
+AWS
+
+# credentialsMode is "Manual"
+$ oc get cloudcredential cluster -o jsonpath --template '{ .spec.credentialsMode }'
+Manual
+
+# serviceAccountIssuer is non empty
+$ oc get authentication cluster -o jsonpath --template='{ .spec.serviceAccountIssuer }'
+abutcher-oidc.s3.us-east-1.amazonaws.com
+```
+
+``. This will operate in one of two ways depending on which other
```

Comment:
I think the backticks here are causing rendering issues

Reply:
they look accidental, fixing.